metering/configuring_metering/metering-common-config-options.adoc

```diff
@@ -82,22 +82,22 @@ spec:
   reporting-operator:
     spec:
       nodeSelector:
-        "node-role.kubernetes.io/infra": "true"
+        "node-role.kubernetes.io/infra": ""
 
   presto:
     spec:
       coordinator:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": ""
       worker:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": ""
   hive:
     spec:
       metastore:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": ""
       server:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": ""
```

Comment:
"node-role.kubernetes.io/infra": "" **<1>**

Reply:
And, below the code block after the closing dashes:

----
<1> Set this field to 'true' if you want to use a nodeSelector to specify a specific pod for your component to run on.

Reply:
Hmm, I'm not sure this still gets the message across that the node label provided in this configuration needs to exist on the set of nodes in order for the metering Pods to become scheduled.

Reply:
It looks like the logging documentation essentially mirrors the kubernetes documentation for node selection: https://docs.openshift.com/container-platform/4.3/logging/cluster-logging-moving-nodes.html

Reply:
Logging note is good. I'll open a PR for this per your request.
https://github.com/openshift/openshift-docs/pull/21627